mandatory/2-function-creation.js

```diff
@@ -16,7 +22,13 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if(num > 100){
+    return 100+"%"
+  }else{
+   return Number(num.toFixed(2))+ "%"
+  }
+}
 
```

Comment:
well done!! I had previously tried using .toFixed() but it didn't work for me. I tested it the way you did and figured out what went wrong. Nice take on this.